Hadrons/Modules/MGuesser/BatchExactDeflation.hpp

```diff
@@ -127,12 +179,19 @@ class BatchExactDeflationGuesser: public LinearFunction<typename Pack::Field>
         for (unsigned int j = si; j < sf; ++j)
         {
             axpy(out[j], 
-                 TensorRemove(innerProduct(epack_.evec[i], in[j]))/epack_.eval[i], 
-                 epack_.evec[i], out[j]);
+                 TensorRemove(innerProduct(epack.evec[i], in[j]))/epack.eval[i], 
+                 epack.evec[i], out[j]);
         }
         t += usecond();
         // performance (STREAM convention): innerProduct 2 reads + axpy 2 reads 1 write = 5 transfers
         LOG(Message) << "projAccumulate: " << t << " us | " << 5.*nIt*lSizeGB << " GB | " << 5.*nIt*lSizeGB/t*1.0e6 << " GB/s" << std::endl;
+    }
+private:
```

Comment:
This is not useful, please remove and use the static function in the main code

Reply:
removed